src/VisualStudio/Core/Impl/SolutionExplorer/DiagnosticItem/CpsDiagnosticItemSourceProvider.cs

```diff
@@ -100,66 +101,61 @@ internal sealed class CpsDiagnosticItemSourceProvider : AttachedCollectionSource
         /// </summary>
         private static string? GetTargetFrameworkMoniker(IVsHierarchyItem item)
```

Comment:
Unrelated to this PR, but the returned string here is actually the target framework "alias", not the moniker.

The moniker looks like `.NETFramework,Version=v4.5.1` while the alias looks like `net451`.

Unfortunately the TFM prefix on the flags is misleading, but it's too late to change that now.